cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/validation/CassandraValidation.java

```diff
@@ -29,7 +29,7 @@
  */
 public class CassandraValidation implements StartupValidation
 {
-    private static final int TIMEOUT_SECONDS = 30;
+    private static final int TIMEOUT_SECONDS = 300;
```

Comment:
should these be configurable?

Reply:
Made timeouts for Sidecar/Cassandra startup validations configurable.